dcs/task.py

```diff
@@ -1077,6 +1082,39 @@ def __eq__(self, other) -> bool:
         return False
 
 
+class CarpetBombing(Task):
+    Id = "CarpetBombing"
+
+    class AttackType(Enum):
+        Carpet = "Carpet",
+
+        def __eq__(self, other: Any) -> bool:
+            if isinstance(other, str):
+                return self.value == other
+            return self == other
+
+    def __init__(self, altitude: int = 2000, altitude_enabled: bool = False,
+                 attack_qty: int = 1, attack_qty_limit: bool = False,
+                 attack_type: AttackType = AttackType.Carpet, carpet_length: int = 500,
```

Comment:
s/`carpet_length`/`pattern_length_ft`/

Reply:
Yes, sorry, I meant to say meters, but was reading the ME when I typed it and mine is also configured for US units.